README.md

```diff
@@ -28,68 +28,23 @@ Features not supported:
 
 ## Wallet API
 
-Constructors:
-
-- `generate([icap])` - create an instance based on a new random key (setting `icap` to true will generate an address suitable for the `ICAP Direct mode`)
-- `generateVanityAddress(pattern)` - create an instance where the address is valid against the supplied pattern (**this will be very slow**)
-- `fromPrivateKey(input)` - create an instance based on a raw private key
-- `fromExtendedPrivateKey(input)` - create an instance based on a BIP32 extended private key (xprv)
-- `fromPublicKey(input, [nonStrict])` - create an instance based on a public key (certain methods will not be available)
-- `fromExtendedPublicKey(input)` - create an instance based on a BIP32 extended public key (xpub)
-- `fromV1(input, password)` - import a wallet (Version 1 of the Ethereum wallet format)
-- `fromV3(input, password, [nonStrict])` - import a wallet (Version 3 of the Ethereum wallet format). Set `nonStrict` true to accept files with mixed-caps.
-- `fromEthSale(input, password)` - import an Ethereum Pre Sale wallet
-
-For the V1, V3 and EthSale formats the input is a JSON serialized string. All these formats require a password.
```

Comment:
@alcuadrado This information gets lost. Can you add to the docs?

Reply:
I think I removed this because the info is now present in the types, so the docs actually have this info. For example https://github.com/ethereumjs/ethereumjs-wallet/pull/98/files#diff-682af32d204096c4ec56fd57a1d07d6cR362

Should I add this as a comment to the class anyway?

Reply:
Added them as `@param` tags